### PR TITLE
Use hostname instead of hostname:port

### PR DESCRIPTION
**Pull request by nakulkar-msft**

### CODE REVIEW

**Review by vibhansa-msft — APPROVED** [review]

**Comment by vibhansa-msft** [comment]
Add changelog to mark the bug fix

File: common/util.go [hunk 1]
```diff
@@ -15,6 +15,6 @@ func VerifyIsURLResolvable(url_string string) (error) {
 		return err
 	}
 
-	_, err = net.LookupIP(url.Host)
+	_, err = net.LookupIP(url.Hostname())
```
Comment (gapra-msft) [inline common/util.go:18]: Is there a unit test for this method we can add some cases to? If not might be a good idea to add a few tests?

**Comment by siminsavani-msft** [comment]
Fixes: https://github.com/Azure/azure-storage-azcopy/issues/2140

File: common/util.go (hunk 1 above)
Comment (siminsavani-msft) [reply common/util.go:18]: +1

**Pull request closed without merging**